army schema work/1 table.sql

```diff
@@ -0,0 +1,42 @@
+use ArmyDB
+go
+
+drop table if exists dbo.Soldier
+go 
+
+create table dbo.Soldier
+(
+FirstName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_FirstName_cannot_be_blank CHECK (FirstName <> ''),
+LastName VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_LastName_cannot_be_blank CHECK (LastName <> ''),
+SSN VARCHAR(10) not null 
+    CONSTRAINT u_Soldier_SSN_must_be_unique unique,
+    CONSTRAINT ck_Soldier_SSN_have_9_digits CHECK (SSN like '%%%%%%%%-%'),
+DateBorn date not null
+    CONSTRAINT ck_Soldier_DateBorn_cannot_be_blank CHECK (DateBorn <> ''),
+City VARCHAR(20) not null
+    CONSTRAINT ck_Soldier_City_cannot_be_blank CHECK (City <> ''),
+EnlistmentDate date not null 
+    CONSTRAINT ck_Soldier_EnlistmentDate_must_be_before_current_date CHECK (EnlistmentDate < GETDATE()),
+    CONSTRAINT ck_Soldier_EnlistmentDate_cannot_be_blank CHECK (EnlistmentDate <> ''),
+MilitaryBranch VARCHAR(12) not null
+    CONSTRAINT ck_Soldier_MilitaryBranch_cannot_be_blank CHECK (MilitaryBranch <> ''),
```

Comment:
-- LL you don't need this not blank constraint bec the other constraint is checking that it can only be one of three things already so that will also ensure that it won't be blank